apps/wallet/src/ui/app/components/accounts/AccountItemApproveConnection.tsx

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { AccountIcon } from '_components/accounts/AccountIcon';
+import { LockUnlockButton } from '_components/accounts/LockUnlockButton';
+import { useUnlockAccount } from '_components/accounts/UnlockAccountContext';
+import { type SerializedUIAccount } from '_src/background/accounts/Account';
+import { useResolveSuiNSName } from '@mysten/core';
+import { CheckFill16 } from '@mysten/icons';
+import { formatAddress } from '@mysten/sui.js/utils';
+import clsx from 'clsx';
+
+interface Props {
+	account: SerializedUIAccount;
+	selected: boolean;
```

Comment:
this could be optional and disabled below, to avoid having to pass something like `selected={false}`

Reply:
added optional